File: C++/Day6/Shapes/Rectangle.cpp
#include "Rectangle.h"
#include <iostream>


Rectangle::Rectangle()
{
    std::cout << "Cons Rectangle \n" ;
}

Rectangle::Rectangle(int d1 ,int d2){Dim1=d1 ; Dim2=d2 ; std::cout << "rectangle cons \n" ;}


/// chain Constractor

Rectangle::~Rectangle()
{
    std::cout << "destructor Rectangle \n";
}

double Rectangle::CalcArea(){
    double res = Dim1 * Dim2;
    std::cout << "Rectangle Area =  "<< res << "\n";
    return res;
}

File: ES6/D03/Task1/Rectangle.js
import { Ploygon } from "./Ploygon.js";

export class Rectangle extends Ploygon{
    
    constructor(d1,d2){
        super(d1, d2)
    }

    toString(){
        console.log( `Area of Rectanle = ${this.Dim1 * this.Dim2} ` )
    }


}